tests/templates/test_subroutines/test_prepselprep.py

```diff
@@ -92,13 +89,10 @@ def manual_circuit(lcu, control):
         qml.AmplitudeEmbedding(qml.math.sqrt(coeffs), normalize=True, pad_with=0, wires=control)
     )
 
-    return qml.state()
```

Comment:
Those are just qfuncs, and there is no need to turn them into QNodes at all, so we can skip the dummy measurement.